minicriu-client.c

```diff
@@ -102,12 +144,295 @@ static int writefile(const char *file, const char *buf, size_t len) {
 	return bytes;
 }
 
+static unsigned long align_up(unsigned long v, unsigned p) {
+	return (v + p - 1) & ~(p - 1);
+}
+
+static int mc_save_core_file() {
+
+	pid_t pid = syscall(SYS_getpid);
+	int phnum = 0;
+
+	// Create Elf header
+	memcpy(ehdr.e_ident, ELFMAG, SELFMAG);
```

Comment:
Yeah, it helps to avoid manual initialization with zeros. [4c267eb](https://github.com/CRaC/minicriu/pull/9/commits/4c267ebc0c199b1ce764ae843f7aedf5e52b2c2e)